packages/api-openfin/src/window.js

```diff
@@ -1,43 +1,80 @@
 let currentWindow = null;
 import MessageService from './message-service';
 
+const optionsMap = {
+  'alwaysOnTop': 'alwaysOnTop',
+  'backgroundColor': 'backgroundColor',
+  'child': 'child',
+  'center': 'defaultCentered',
+  'frame': 'frame',
+  'hasShadow': 'shadow',
+  'height': 'defaultHeight',
+  'maxHeight': 'maxHeight',
+  'maximizable': 'maximizable',
+  'maxWidth': 'maxWidth',
+  'minHeight': 'minHeight',
+  'minimizable': 'minimizable',
+  'minWidth': 'minWidth',
+  'resizable': 'resizable',
+  'show': 'autoShow',
+  'skipTaskbar': 'showTaskbarIcon',
+  'transparent': 'opacity',
+  'width': 'defaultWidth',
+  'x': 'defaultLeft',
+  'y': 'defaultTop'
+};
+
 class Window {
   constructor(...args) {
     if (args.length === 0) {
       this.innerWindow = fin.desktop.Window.getCurrent();
     } else {
-      const [url, name, features] = args;
+      const [url, name, options] = args;
 
       let newWindow;
       const handleError = (error) => console.error('Error creating window: ' + error);
 
-      const windowOptions = features || {};
-      if (windowOptions.transparent) {
+      const convertedOptions = {};
+
+      Object.keys(optionsMap).forEach((key) => {
+        const openfinOption = optionsMap[key];
+        convertedOptions[openfinOption] = options[key];
+      });
+
+      const mergedOptions = Object.assign(
+        {},
+        convertedOptions,
+        options
+      );
+
+      if (options.transparent) {
         // OpenFin needs opacity between 1 (not transparent) and 0 (fully transparent)
-        windowOptions.opacity = features.transparent === true ? 0 : 1;
+        mergedOptions.opacity = options.transparent === true ? 0 : 1;
+      }
+      if (options.skipTaskbar) {
+        mergedOptions.showTaskbarIcon = !options.skipTaskbar;
       }
 
-      if (windowOptions.child) {
-        const options = Object.assign(
+      if (mergedOptions.child) {
+        const childOptions = Object.assign(
           {},
-          windowOptions,
+          mergedOptions,
           {
             name,
             url
           }
         );
 
-        newWindow = new fin.desktop.Window(options, () => newWindow.show(), handleError);
+        newWindow = new fin.desktop.Window(childOptions, null, handleError);
```

Comment:
We could just have `newWindow = new fin.desktop.Window(childOptions);`. We need to add window success/error callbacks at some point anyway